src/TLuaInterpreterMapper.cpp

```diff
@@ -1040,6 +1042,7 @@ int TLuaInterpreter::createMapper(lua_State* L)
         return warnArgumentValue(L, __func__, message);
     }
 
+    host.mpMap->update();
```

Comment:
I would have thought that creating the map would cause it to be drawn - so is this really needed here?

Reply:
Updated.